### ISSUE TITLE
Need to pass the value to anonymous function in Select.js

### ISSUE THREAD

**Issue by Henry0422** [foam-framework/foam2#1161]
The medal-color drop-down menu in the Olympic demo project (foam2/src/com/google/foam/demos/olympics/index.html) is showing **3 Bronze options** 

This issue is associate with e6804340cbc4e6a12d4178c2a7db035c07f2d6f9. In `Select.js`, for the following part in the for loop,  only the last `value` be passed to the anonymous function during the iteration. 
``` javascript
var c = choices[i];
var value = c[1];
let e = self.E('option').attrs({
    value: i,
    selected: self.data === i ? true : undefined
}).add(value);
if ( value.indexOf('  ') != -1 ) {
    // Hack to display spaces as nbsp's
    e.onload.sub(function() {
    e.el().innerHTML = value.replace(/ /g, '&nbsp;');
    });
}
```

**Soluction**: change `var value = c[1];` to `let value = c[1];`
or using closure in order to pass the context of `e` & `value`
```javascript
var c = choices[i];
var value = c[1];
var e = self.E('option').attrs({
value: i,
selected: self.data === i ? true : undefined
}).add(value);

if ( value.indexOf('  ') != -1 ) {
    var createFunc = function(e, value) {
        return function() {
        e.el().innerHTML = value.replace(/ /g, '&nbsp;');
        }
    }
    // Hack to display spaces as nbsp's
    e.onload.sub(createFunc(e, value));
}
```

**Comment by kgrgreer** [comment]
Just change 'var value' to 'let value'.

**Comment by Henry0422** [comment]
Fixed in commit 5b45130106824166f5841e96b81b971ed8d439eb
